notes/eligibility-summary.vbs

```diff
@@ -23444,6 +23444,82 @@ class stat_detail
 		stat_mont_form_recvd_date = replace(stat_mont_form_recvd_date, " ", "/")
 		If stat_mont_form_recvd_date = "__/__/__" Then stat_mont_form_recvd_date = ""
 
+		'Herehere
+		'TODO: Is this okay placement? 
+		'TODO: Why does this only work if inside the sub?
+		'TODO: Why does it take so long to initially load the script?
+		'TODO: Do we like how it double checks the value or do we not want to read the statuses twice (once above and once within the error handling). Could restructure:
```

Comment:
No need to read statuses twice, they won't change. We should not include an exit here though. This is information gathering, we want to do things with that information later.